com.fyralabs.Exponential.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Name=Exponential
+Comment=A purpose-built tool for planning and building products
+Icon=com.fyralabs.Exponential
+Exec=/app/bin/com.fyralabs.Exponential %U
```

Comment:
this is never a path.

Reply:
Shouldn't this install to that path:
https://github.com/flathub/flathub/pull/5614/files#diff-72eb0b8bf04f6d2d4464f1f57a3d341ce93fae36ae7408a2fcaa84b2ccf7355dR26

Reply:
Exec does not need a path, `/app/bin` is already in $PATH inside the sandbox, it is automatically located.